fantomas.sln

```diff
@@ -20,15 +20,30 @@ Project("{6EC3EE1D-3C4E-46DD-8F32-0CC8E7565705}") = "Fantomas.Benchmarks", "src\
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "docs", "docs", "{29F22904-C9E0-40AA-B971-9AE377C17F59}"
 	ProjectSection(SolutionItems) = preProject
-		docs\Documentation.md = docs\Documentation.md
-		docs\FormattingConventions.md = docs\FormattingConventions.md
```

Comment:
Removed these from the solution since they no longer exist.

Reply:
Thanks!